locale/af/LC_MESSAGES/django.po

```diff
@@ -711,6 +718,11 @@ msgstr "Nog nie gegradeer nie"
 msgid "<b>{0}</b> users"
 msgstr "<b>{0}</b> gebruikers"
 
+#: src/olympia/addons/templates/addons/persona_preview.html
+#, fuzzy
+msgid "{0} Daily Users"
+msgstr "Daaglikse gebruikers"
```

Comment:
Heh, good point. It's a useful reminder for me to turn off fuzzy for the disco pane now we're past that change that broke the keys.